src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+}
+
+const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const numberOfPages = Math.ceil(total / perPage);
+  const pages = Array.from({ length: numberOfPages }, (_, i) => i + 1);
+
+  const handlePrevClick = (
+    currentEvent: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    currentEvent.preventDefault();
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNextClick = (
+    currentEvent: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    currentEvent.preventDefault();
+    if (currentPage < numberOfPages) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
```

Comment:
what value return this expression
`currentPage === 1`
Do you need ternary here?